tests/acceptance/course-page/try-other-language-test.js

```diff
@@ -36,7 +36,7 @@ module('Acceptance | course-page | try-other-language', function (hooks) {
       'fetch repositories (course page)',
       'fetch leaderboard entries (course page)',
       'fetch hints (course page)',
-      // 'fetch language guide (course page)',
+      'fetch language guide (course page)',
```

Comment:
Hmm, weren't we already loading this from the current card on the page? 🤔 @andy1li do you know why this only has to be uncommented now?

Reply:
@rohitpaulk I needed to uncomment three of them, otherwise the tests won't pass.

Context:

These tests were set up on the second stage of redis. It seems that language guides had been loaded before recent changes.